packit/config/package_config_validator.py

```diff
@@ -52,6 +55,18 @@ def validate(self) -> str:
         except PackitConfigException as e:
             return str(e)
 
+        specfile_path = self.content.get("specfile_path", None)
+        if specfile_path and not Path(specfile_path).is_file():
+            logger.warning(
+                f"The spec file you defined {specfile_path} is not \
+                    present in the repository. If it's being generated \
+                    dynamically, you can verify the functionality by \
+                    running `packit srpm` to create an SRPM \
+                    from the current checkout. If it's not being generated, \
+                    please make sure the path is correct and \
+                    the file is present."
```

Comment:
This needs to be wrapped differently. The current output is somewhat unreadable :)

```
$ packit-dev validate-config
2021-08-18 16:15:34.287 package_config_validator.py WARNING The spec file you defined curl.spec is not                     present in the repository. If it's being generated                     dynamically, you can verify the functionality by                     running `packit srpm` to create an SRPM                     from the current checkout. If it's not being generated,                     please make sure the path is correct and                     the file is present.
2021-08-18 16:15:34.287 validate_config.py INFO   .packit.yaml is valid and ready to be used
```

Reply:
Whoops sorry, fixed it :)